# zixue-file
just messing around 
what did i do here
1. I set up my local path "zixue" in the folder python stuff
2. I created the repo on github online
3. You open up the desktop github and under file the the top left corner you clone the pytho: "note that mistakes take up different form, what you did here is you flipped the order of repo and local
4. now we are trying to sychn some changes 

now try to unprivate the file
"^ the message above can be tracked in "history" 
Go to the "settings>danger zone>change the visibility"

how to return to the historical versions? 
